### ISSUE TITLE
Method CanInsert..Property of the controllers always return true

### ISSUE THREAD

**Issue by gekiss** [skoruba/Duende.IdentityServer.Admin#235]
### Describe the bug
The methods:
CanInsertApiResourceProperty of the ApiResourcesController
CanInsertApiScopeProperty of the ApiScopesController
CanInsertIdentityResourceProperty of the IdentityResourcesController
always return true value

#### Create new Api Resource
``` 
curl -X 'POST' \
  'https://localhost:44302/api/ApiResources' \
  -H 'accept: */*' \
  -H 'Authorization: Bearer eyJ...' \
  -H 'Content-Type: application/json' \
  -d '{
  "id": 0,
  "name": "string",
  "displayName": "string",
  "description": "string",
  "enabled": true,
  "showInDiscoveryDocument": true,
  "requireResourceIndicator": true,
  "userClaims": [
    "string"
  ],
  "allowedAccessTokenSigningAlgorithms": [
    "string"
  ],
  "scopes": [
    "string"
  ]
}'
```
```
{
  "id": 5,
  "name": "string",
  "displayName": "string",
  "description": "string",
  "enabled": true,
  "showInDiscoveryDocument": true,
  "requireResourceIndicator": true,
  "userClaims": [
    "string"
  ],
  "allowedAccessTokenSigningAlgorithms": [
    "string"
  ],
  "scopes": [
    "string"
  ]
}
```
#### Validate the property with key `key` exists for resource, return *true* as expected
```
url -X 'GET' \
  'https://localhost:44302/api/ApiResources/CanInsertApiResourceProperty?id=5&key=key' \
  -H 'accept: application/json' \
  -H 'Authorization: Bearer eyJ...' \
```
```
true
```
#### Add the property with key `key` and value `value` to the resource
```
curl -X 'POST' \
  'https://localhost:44302/api/ApiResources/5/Properties' \
  -H 'accept: */*' \
  -H 'Authorization: Bearer eyJ...' \
  -H 'Content-Type: application/json' \
  -d '{
  "key": "key",
  "value": "value"
}'
```
```
{
  "id": 1,
  "key": "key",
  "value": "value"
}
```
#### Validate the property with key `key` exists for resource, return *true*, but expected *false*
```
url -X 'GET' \
  'https://localhost:44302/api/ApiResources/CanInsertApiResourceProperty?id=5&key=key' \
  -H 'accept: application/json' \
  -H 'Authorization: Bearer eyJ...' \
```
```
true 
```

**Comment by skoruba** [comment]
I will check it, thanks!

**Comment by skoruba** [comment]
There is a bug, I will release the fix in 2.4.0 soon.
Thanks for letting me know.

**Comment by skoruba** [comment]
Released in 2.4.0. 
Thanks!
